rdr_service/tools/tool_libs/resource_tool.py

```diff
@@ -1426,24 +1264,24 @@ def update_argument(p, dest, help=None):  # pylint: disable=redefined-builtin
             if a.dest == dest:
                 a.help = help
 
-    def argument_conflict(args_, ids_, choices=()):
-        """ Check if common arguments conflict """
-        if args_.table and args_.all_tables:
-            _logger.error("Arguments 'table' and 'all-tables' conflict.")
-            return True
-        elif args_.all_tables and args_.from_file:
-            _logger.error("Argument 'from-file' cannot be used with 'all-tables', only with 'table'")
-            return True
-        elif args_.id and ids_:
-            _logger.error("Argument 'from-file' cannot be used if a single 'id' was also specified")
-            return True
-        elif ids_ and not args_.table:
-            _logger.error("Argument 'from-file' was provided without a specified 'table' ")
-            return True
-        if args_.table and args_.table not in choices:
-            _logger.error(f"Argument 'table' value '{args_.table}' is invalid, possible values are:\n   {choices}.")
-            return True
-        return False
+    # def argument_conflict(args_, ids_, choices=()):
```

Comment:
Should we remove this too??

Reply:
> Should we remove this too??

@dtharpeuno This is a PDR-specific tool and I want to leave that code there in case the remaining functionality needs to re-enable it.